ex11/part4/src/colorbox.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+// import {useState} from 'react';
+
+export default function colorbox(prop) {
+
+    return (
+        <input type={"text"} value={prop.defaultColor} onChange={prop.toUpdate} />
```

Comment:
You can use here `<input type="color"` (also, no need for expression brackets `{` around text attributes)